rules.html

```diff
@@ -3,36 +3,32 @@
 
 <head>
     <title>Rules</title>
+    <link href="styles.css" rel="stylesheet" type="text/css" />
 </head>
 
 <body>
     <div id="rules">
         <h4>Rule and Information:</h4>
         <ul>
-            <li>This is a modified game of Uno, matching color or card value</li>
-            <li>Cards with value 10 are Draw 2</li>
-            <li>Cards with value 11 are Reverse Play Order</li>
-            <li>Cards with value 12 are Skip next Player's Turn</li>
-            <li>
-                Cards can be Red, Green, Blue, or Yellow. 0-9, +2, switch directions,
-                and skips for each color
-            </li>
-            <li>
-                To play a card simply enter the index of the card starting at the
-                leftmost card at 0 (0, 1, 2, 3, ... )
-            </li>
-            <li>
-                You do not have to directly call Uno, to win you simply play your
-                final card
-            </li>
-            <li>
-                If you do not have a valid card to play you must draw until you get
-                one you can play
-            </li>
-            <li>There are no special rules in this version (e.g swap decks)</li>
-            <li>Bots play by the same rules as you do</li>
-            <li>Bots require some time to "think"</li>
+            <li style=" margin-bottom: 5px;">This is a modified game of Uno, matching color or card value.</li>
```

Comment:
In styles.css, you can do this:
`
li {
    margin-bottom: 5px;
}
`
Then replace all of these
`<li style="....">`
with just
`<li>`